teckers/teckers/Authentification.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  Auth.swift
+//  teckers
+//
+//  Created by Ricardo Granja on 8/8/19.
+//  Copyright © 2019 Teckers. All rights reserved.
+//
+
+import Foundation
+import Firebase
+import GoogleSignIn
+import Alamofire
+
+class Authentification: NSObject, GIDSignInDelegate  {
+    
+    override init(){
+        super.init()
+        GIDSignIn.sharedInstance().clientID = FirebaseApp.app()?.options.clientID
+        GIDSignIn.sharedInstance().delegate = self
+    }
+    
+    static func singOut() -> (flag: Bool, error: String) {
```

Comment:
why is this func static and login function is not static?

Reply:
Login is not static because it does not allow us to modify the delegate, and the same function is called by the delegate when the button is pressed to enter (Sign In) t the Sign Out method It is a method that we build, the delegates do not come.